app_server/app/adapters/socket/Room.js

```diff
@@ -75,4 +75,21 @@ Class.removeAllClients = function() {
   this.__clients = {};
 };
 
+Class.getNumberOfUsers = function() {
+  let users = {};
+  let count = 0;
+  for (var socketId in this.getClients()) {
+    let userId = this.getClient(socketId).getUserId(); 
+    if (!users[userId]) {
+      users[userId] = true;
+      count++;
+    }
+  }
+  return count;
```

Comment:
Would a functional style work better here?
Start off with a `Object.keys(this.getClients()).filter()`, then run a `forEach()` and `length`.
Either way's fine, though.